DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -13,7 +13,7 @@
     destinations = cms.untracked.vstring('cout')
 )
 
-live=True
+live=False
```

Comment:
Are you sure live mode has to be disabled?

Reply:
Hi Andrius. Sorry for the late answer. I miss your messages. Well, this PR was not meant to be put in "production" since 10_2_5 is closed. It was more to have a bookkeeping of the filter we added for high pu. In that sense, having it True or not doesn't matter ?

Reply:
Hi @jandrea, ok, I see. You also mentioned in the main comments section that there is no plan to forward-port this PR. Could you please close it in such case?